bin/drug-screening-workflow/dose-response-metric-workflow.cwl

```diff
@@ -0,0 +1,72 @@
+class: Workflow
+cwlVersion: v1.0
+id: dose_response_metrics_from_synapse
+label: dose-response-metrics-from-synapse
+$namespaces:
+  sbg: 'https://www.sevenbridges.com/'
+inputs:
+  - id: synapse_config
+    type: File
+    'sbg:x': -52.59514236450195
```

Comment:
plz remove the rabix coordinates.

Reply:
This was brought up at the last WG meeting. It's quite useful to have the coords for loading into Rabix, but also equally annoying that they aren't in a separate config file.

Reply:
also annoying - inadvertently opening the file in Rabix instead of a text editor sticks them back in there.